networkx/algorithms/centrality/voterank_alg.py

```diff
@@ -90,3 +90,92 @@ def voterank(G, number_of_nodes=None):
             vote_rank[nbr][1] -= 1 / avgDegree
             vote_rank[nbr][1] = max(vote_rank[nbr][1], 0)
     return influential_nodes
+
+
+def vote_rank(G, number_of_nodes=None):
+    """Select a list of influential nodes in a graph using VoteRank algorithm
+
+    VoteRank [1]_ computes a ranking of the nodes in a graph G based on a
+    voting scheme. With VoteRank, all nodes vote for each of its in-neighbours
+    and the node with the highest votes is elected iteratively. The voting
+    ability of out-neighbors of elected nodes is decreased in subsequent turns.
+
+    Parameters
+    ----------
+    G : graph
+        A NetworkX graph.
+
+    number_of_nodes : integer, optional
+        Number of ranked nodes to extract (default all nodes).
+
+    Returns
+    -------
+    voterank : dict
+        Unordered dictionary with nodes as a key and vote as a value.
```

Comment:
I think this is an ordered dict, right?  For Python 3.6 or later dicts are ordered in the order the keys are added. If it is ordered, the order should probably be tested.